src/components/header.css

```
@@ -0,0 +1,97 @@
+
+.dailyFundingLink {
+  display: flex;
+  align-items: center;
+  gap: 6px;
+}
+
+.dailyFundingLink span {
+  font-size: 18px;
+  letter-spacing: -1.08px;
+  font-weight: 500;
+  font-display: block;
```

Stylelint (17.4.0)

[error] Unexpected unknown property "font-display" (property-no-unknown)

(property-no-unknown)